pages/api/account/manage/link/[[...data]].js

```diff
@@ -55,6 +54,18 @@ export async function getLinkApi(username, id) {
   return JSON.parse(JSON.stringify(getLink));
 }
 
+export async function getGroupLinkApi(username) {
+  await connectMongo();
+  let getGroupLink = [];
+  getGroupLink = await Link.aggregate([
+    { $match: { username } },
+    { $group: { _id: null, groups: { $addToSet: "$group" } } },
+    { $project: { _id: 0, groups: 1 } },
+  ]).exec();
+  getGroupLink = getGroupLink[0].groups.filter((item) => item !== "");
```

Comment:
Does this need a check for `.length`?

Reply:
I don't think this need a check for length the getGroupLink returns a a array with group as it's object
```js
getGroupLink = [ { groups : [""] } ];
```
even if the query don't find any goup this will still return the data in the format i mentioned above.

Reply:
You can verify this by logging the values.

Reply:
ok thank you 👍